Assets/Scripts/UI/Menu's/MainMenuController.cs

```diff
@@ -0,0 +1,21 @@
+using UnityEngine.SceneManagement;
+using UnityEngine;
+
+public sealed class MainMenuController : MenuController
+{
+    [SerializeField] private string mainScene;
+    [SerializeField] private GameObject activeExplanationMenu;
+    [SerializeField] private GameObject activeMainMenu;
```

Comment:
```suggestion
    [SerializeField] private GameObject explanationMenu;
    [SerializeField] private GameObject mainMenu;
```
Zelfde feedback als feedback 1.